### PR TITLE
PivotGrid.CustomStore - Samples for Angular/Vue/React

### PR DESCRIPTION
**Pull request by novsstation**

### CODE REVIEW

File: concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md [hunk 1]
```diff
@@ -54,6 +55,396 @@ The example below shows how to implement the **load** function. Note that in thi
...
+                params += `${i}=${JSON.stringify(loadOptions[i])}&`;
+              }
+            });
+            params = params.slice(0, -1);
+            return fetch(
+              `https://js.devexpress.com/Demos/WidgetsGalleryDataService/api/Sales/orders${params}`
+            )
+              .then((response) => response.json())
+              .then((result) => {
+                return new Promise((resolve) => {
```
Comment (IgnatovDan) [inline concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md]: Let's handle 'response.ok' in our demo code:

    if (!response.ok) {
      throw new Error('Network response was not ok');
    }

https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch#checking_that_the_fetch_was_successful

File: concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md [hunk 2]
```diff
@@ -54,6 +55,396 @@ The example below shows how to implement the **load** function. Note that in thi
...
+        if (i in loadOptions && isNotEmpty(loadOptions[i])) {
+          params += `${i}=${JSON.stringify(loadOptions[i])}&`;
+        }
+      });
+      params = params.slice(0, -1);
+      return fetch(
+        `https://js.devexpress.com/Demos/WidgetsGalleryDataService/api/Sales/orders${params}`
+      )
+        .then((response) => response.json())
+        .then((result) => {
```
Comment (IgnatovDan) [inline concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md]: Let's handle 'response.ok' in our demo code

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md (hunk 1 above)
Comment (novsstation) [reply concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md]: Yep. Great idea.
Fixed - https://github.com/DevExpress/devextreme-documentation/pull/2316/commits/d596d84c18a3c9b67b9c9ec86cf0053b5431d0e8

File: concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md (hunk 2 above)
Comment (novsstation) [reply concepts/05 UI Components/PivotGrid/035 Use CustomStore/20 Remote Operations.md]: Yep. Great idea.
Fixed - https://github.com/DevExpress/devextreme-documentation/pull/2316/commits/d596d84c18a3c9b67b9c9ec86cf0053b5431d0e8

**Review by RomanTsukanov — APPROVED** [review]

**Pull request merged**